domains/yean.json

```diff
@@ -0,0 +1,12 @@
+{
+    "owner": {
+        "username": "joshex150",
+        "email": "joshex150@gmail.com"
+    },
+    "record": {
+    "URL": "https://yean-tech.vercel.app",
+    "TXT": [
+      "vc-domain-verify=yean.is-a.dev,c671dbf711ab7400712a"
+    ]
```

Comment:
Invalid record, as the Vercel verification should be separate file and no Vercel records were found.